pagespeed/envoy/envoy_cluster_manager.cc

```diff
@@ -0,0 +1,104 @@
+#include "envoy_cluster_manager.h"
+
+#include <sys/file.h>
```

Comment:
is this include needed here?

Reply:
yes, it was a mistake. Although we used cluster.yaml file in this file, it was just a string reference. Actual file read operation was not getting performed.
Code pushed in https://github.com/apache/incubator-pagespeed-mod/pull/1969/commits/05e80e98faba31ff5a94f1244b1833fa88a1fae4